index.html

```diff
@@ -0,0 +1,163 @@
+<!Doctype html>
+<html lang="en">
+    <head>
+        <title>Portfolio page</title>
+        <link rel ="stylesheet" type="text/css" href="normalise.css">
+        <meta name="viewport contents" content="device-width, initial-scale=1.0">
+        <link rel ="stylesheet" type="text/css" href="portfolio-style.css">
+    </head>
+    <body>
+        <header>
+            <section class="nav-bar">
+                <div class="name">
+                    <h3>Matthew Hammond</h3>
+                </div>
+                <div class="nav-links">
+                    <a href="#bio"><button type="button">Bio</button></a>
+                    <a href="#portfolio"><button type="button">Portfolio</button></a>
+                    <a href="#contact"><button type="button">Contact me</button></a>
+                </div>
+            </section>
+
+            <section class="landing">
+                <div class="landing-container">
+                    <img class="landing-image" src="./Assets/aerial.jpg" alt="landing-photograph">
+                    <div class="hero">
+                        <h1>Matthew Hammond</h1>
+                    </div>
+                    <div class="hero-description">
+                         <h1>Full-stack developer</h1>
+                    </div>
+                </div>
+            </section>
+            <!-------------------------------------------->
```

Comment:
Comments are ok, still, but they should be adding something to the code and used consistently.

There isn't one for the nav bar and this won't tell you anything useful next week